Source/UIKit additions/SVGKFastImageView.h

```diff
@@ -40,14 +38,11 @@
  can invent a workaround (or Apple fixes their bug), it's best to warn developers that their SVG will NOT render
  correctly
  */
-+(BOOL) svgImageHasNoGradients:(SVGKImage*) image;
++(BOOL)svgImageHasNoGradients:(SVGKImage*)image;
 
-/** Apple has a bug in CALayer where their renderInContext: method does not respect Apple's own mask layers.
- 
- This is required to render SVGGradientElement's, and it is NOT a bug in SVGKit - it's in Apple's code. Until we
- can invent a workaround (or Apple fixes their bug), it's best to warn developers that their SVG will NOT render
- correctly
+/** The text implementation on OS X is different between CALayers and NSViews. If the CALayer is made in
+ SVGKLayeredImageView, it renders right-side up. Otherwise, it is upside-down.
  */
-+(BOOL) svgElementAndDescendentsHaveNoGradients:(SVGElement*) element;
```

Comment:
Ops..I think this API is un-used. And here is one `svgImageHasNoGradients`.

Anyway, this may break old API (at least we should deprecate but not remove it). I will keep this.

Reply:
One question, should't these API be wrapped inside `TEMPORARY_WARNING_FOR_APPLES_BROKEN_RENDERINCONTEXT_METHOD` macro ?

If someday we change the .m files value to 0, the public header still contains this define (but no implementation), it may crash because of unrecognized selector.